From 67e76911baa69cb02401916c6ffb430f73c5e799 Mon Sep 17 00:00:00 2001
Message-ID: <67e76911baa69cb02401916c6ffb430f73c5e799.1696804840.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1696804840.git.sharlatanus@gmail.com>
References: <cover.1696804840.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 5 Oct 2023 02:37:43 +0100
Subject: [PATCH 09/20] gnu: python-drizzle: Update package style.

* gnu/packages/astronomy.scm (python-drizzle): Update package style,
indent and align regions.
[arguments]{test-flags}: Remove them as all tests are passed
successfully.
[native-inputs]: Remove python-coverage, python-pytest-cov.
[description]: Indent region.
---
 gnu/packages/astronomy.scm | 63 ++++++++++++++++++--------------------
 1 file changed, 29 insertions(+), 34 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 2569cedd02..e7ee63dcb8 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -1858,47 +1858,42 @@ (define-public python-drizzle
   (package
     (name "python-drizzle")
     (version "1.14.3")
-    (source (origin
-              (method git-fetch) ;PyPi doesn't have the test data sets
-              (uri (git-reference
-                    (url "https://github.com/spacetelescope/drizzle")
-                    (commit version)))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "04gncwg76yivcaq7rwgsw5k8w4h3c4wcrjcamb53h0d5s820z7dl"))))
+    (source
+     (origin
+       (method git-fetch) ;PyPi doesn't have the test data sets
+       (uri (git-reference
+             (url "https://github.com/spacetelescope/drizzle")
+             (commit version)))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "04gncwg76yivcaq7rwgsw5k8w4h3c4wcrjcamb53h0d5s820z7dl"))))
     (build-system pyproject-build-system)
     (arguments
      (list
-      ;; XXX: 2 of 26 tests failed with AssertionError, disable them for now.
-      ;; Consider mention it in upstream.
-      #:test-flags #~(list "-k"
-                           (string-append "not test_square_with_point"
-                                          " and not test_square_with_grid"))
-      #:phases #~(modify-phases %standard-phases
-                   (add-before 'build 'set-env-version
-                     (lambda _
-                       (setenv "SETUPTOOLS_SCM_PRETEND_VERSION"
-                               #$version)))
-                   (add-before 'check 'build-extensions
-                     (lambda _
-                       ;; Cython extensions have to be built before running
-                       ;; the tests.
-                       (invoke "python" "setup.py" "build_ext" "--inplace"))))))
-    (propagated-inputs (list python-astropy python-numpy))
-    (native-inputs (list python-coverage python-flake8 python-pytest
-                         python-pytest-cov python-setuptools-scm))
+      #:phases
+      #~(modify-phases %standard-phases
+          (add-before 'build 'set-env-version
+            (lambda _
+              (setenv "SETUPTOOLS_SCM_PRETEND_VERSION" #$version)))
+          (add-before 'check 'build-extensions
+            (lambda _
+              ;; Cython extensions have to be built before running the tests.
+              (invoke "python" "setup.py" "build_ext" "--inplace"))))))
+    (propagated-inputs
+     (list python-astropy python-numpy))
+    (native-inputs
+     (list python-flake8 python-pytest python-setuptools-scm))
     (home-page "https://github.com/spacetelescope/drizzle")
     (synopsis
      "Astronomical tool for combining dithered images into a single image")
     (description
-     "The drizzle library is a Python package for combining dithered images into
-a single image.  This library is derived from code used in DrizzlePac.  Like
-DrizzlePac, most of the code is implemented in the C language.  The biggest
-change from DrizzlePac is that this code passes an array that maps the input to
-output image into the C code, while the DrizzlePac code computes the mapping by
-using a Python callback.  Switching to using an array allowed the code to be
-greatly simplified.")
+     "The drizzle library is a Python package for combining dithered images
+into a single image.  This library is derived from code used in DrizzlePac.
+Like DrizzlePac, most of the code is implemented in the C language.  The
+biggest change from DrizzlePac is that this code passes an array that maps the
+input to output image into the C code, while the DrizzlePac code computes the
+mapping by using a Python callback.  Switching to using an array allowed the
+code to be greatly simplified.")
     (license license:bsd-3)))
 
 (define-public python-ephem
-- 
2.41.0

